base/permuteddimsarray.jl

```diff
@@ -108,11 +108,18 @@ julia> permutedims(A, [3, 2, 1])
  6  8
 ```
 """
-function Base.permutedims(A::AbstractArray, perm)
+function Base.permutedims(A::AbstractArray, perm = (2,1))
```

Comment:
And likewise here re. confining the default `(2, 1)` permutation to two-dimensional objects? :)